db/migrate/20210601075343_add_more_things_to_movies.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddMoreThingsToMovies < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :movies, :age_type
+    add_column :movies, :adult_only, :boolean
```

Comment:
I would consider creating something like `:movies, :restriction, :integer` instead of boolean, because some movies may have restriction age different than 18 like 16, 14 🤔

Reply:
Oh right, I didn't think about it :d change on way :D